src/wasm/wasm-validator.cpp

```diff
@@ -2313,6 +2313,7 @@ void FunctionValidator::visitFunction(Function* curr) {
   for (const auto& var : curr->vars) {
     features |= var.getFeatures();
     shouldBeTrue(var.isConcrete(), curr, "vars must be concretely typed");
+    // TODO: check for nullability
```

Comment:
We should enforce that locals are non-nullable now, as well as enforcing that types in block signatures and function signatures are non-nullable.

Reply:
Why should locals be non-nullable? Shouldn't they be rather nullable, given that they need a null initialization value?